src/sentry/scripts/utils/locking/delete_lock.lua

```diff
@@ -0,0 +1,12 @@
+local key = KEYS[1]
+local uuid = ARGV[1]
+
+local value = redis.call('GET', key)
+if not value then
+    return redis.error_reply(string.format("No lock at key exists at key: %s", key))
+elseif value ~= uuid then
+    return redis.error_reply(string.format("Lock at %s was set by %s, and cannot be released by %s.", key, value, uuid))
+else
+    redis.call('DEL', key)
```

Comment:
Is it possible at all for this to error? Normally you'd want to do `return redis.call('DEL', key)` instead, but now we're not even checking the response value and assuming it will always exist.